scripts/build-setup.sh

```diff
@@ -273,8 +264,31 @@ if run_step "8"; then
     popd
 fi
 
-# do misc. cleanup for a "clean" git status
 if run_step "10"; then
+    # install circt into conda
+    if run_step "1"; then
+        PREFIX=$CONDA_PREFIX/$TOOLCHAIN_TYPE
```

Comment:
I don't like this being in $RISCV but this needs to get fixed quickly. We should keep track of this and properly fix it later.